cl/corpus_importer/management/commands/update_casenames_wl_dataset.py

```diff
@@ -111,7 +111,7 @@ def parse_date(date_str: str) -> date | None:
     return None
 
 
-def make_citations(cite1, cite2) -> list[FullCaseCitation]:
+def make_citations(cite1: str, cite2: str) -> list[FullCaseCitation]:
```

Comment:
instead of returning full case citation - why not bring back the dictionary of volume page and reporter here so you dont need to remove dates etc later

Reply:
i did it this way because to use the add_citations_to_cluster function it needs a list of strings, and if we return the citations as a dict, we need to do something like: f"{cite.get("volume")} {cite.get("reporter")} {cite.get("volume")}" to get the full citation but we can get the same result using the FullCaseCitation corrected_citation()  method.

I also realized that the reporter returned in the dictionary is not the corrected reporter, for example: if you pass "276 Cal.Rptr. 272" to get_citations function, .groups will return {'volume': '276', 'reporter': 'Cal.Rptr.', 'page': '272'}

In the db we have that citation stored with "Cal. Rptr.", not "Cal.Rptr.", when you try to search for a Citation object using the matched reporter from the dict but not the corrected reporter it will not return any results. I tested the original approach using the dict and couldn´t match a lot of the rows, but changinq the queries to use the corrected_reporter() method from FullCaseCitation fixed that.